test/Microsoft.ML.Benchmarks/Harness/Configs.cs

```diff
@@ -0,0 +1,50 @@
+﻿using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Toolchains;
+using BenchmarkDotNet.Toolchains.CsProj;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+using Microsoft.ML.Benchmarks.Harness;
+
+namespace Microsoft.ML.Benchmarks
+{
+    public class RecommendedConfig : ManualConfig
+    {
+        public RecommendedConfig()
+        {
+            Add(DefaultConfig.Instance
+                .With(GetJobDefinition().With(CreateToolchain()))
+                .With(new ExtraMetricColumn())
+                .With(MemoryDiagnoser.Default));
+
+            UnionRule = ConfigUnionRule.AlwaysUseLocal;
+        }
+
+        protected virtual Job GetJobDefinition()
+            => Job.Default
+                .WithWarmupCount(1) // ML.NET benchmarks are typically CPU-heavy benchmarks, 1 warmup is usually enough
+                .WithMaxIterationCount(20);
```

Comment:
@adamsitnik I think we should make the default configuration as the train config as most of the tests are running in the train configuration ?

Reply:
@Anipik thank you for the suggestion! The `TrainConfig` is a very specific config, I would prefer to not make it a default one